documentation/docs/16-seo.md

```diff
@@ -82,8 +82,40 @@ export async function get() {
 
 #### AMP
 
-An unfortunate reality of modern web development is that it is sometimes necessary to create an [Accelerated Mobile Pages (AMP)](https://amp.dev/) version of your site. In SvelteKit this can be done by setting the [`amp`](/docs/configuration#amp) config option, which has the following effects:
+An unfortunate reality of modern web development is that it is sometimes necessary to create an [Accelerated Mobile Pages (AMP)](https://amp.dev/) version of your site. In SvelteKit this can be done by enforcing the following [configuration](/docs/configuration) options...
 
-- Client-side JavaScript, including the router, is disabled
-- Styles are concatenated into `<style amp-custom>`, and the [AMP boilerplate](https://amp.dev/boilerplate/) is injected
-- In development, requests are checked against the [AMP validator](https://validator.ampproject.org/) so you get early warning of any errors
+```js
+/// file: svelte.config.js
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	kit: {
+		// the combination of these options
+		// disables JavaScript
+		browser: {
+			hydrate: false,
+			router: false
+		},
+
+		// since <link rel="stylesheet"> isn't
+		// allowed, inline all styles
+		inlineStyleThreshold: Infinity
+	}
+};
+
+export default config;
+```
+
+...and transforming the HTML using `transformPage` along with `amp.transform` imported from `@sveltejs/kit/amp`:
```

Comment:
```suggestion
...and transforming the HTML using `transformPage` along with `transform` imported from `@sveltejs/kit/amp`:
```

I'd just refer to this as `transform`, since that's the name of the export.